testing/request-bank.js

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
this code runs in `node` should not be in this `testing/` folder, which so far are all run in browser.

you might move it to `tasks/e2e/`

Reply:
I was trying to unify the two `RequestBank`s, but just realized it was not too helpful to the end.